Segment-Mixpanel.podspec

```diff
@@ -21,6 +21,6 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics', '~> 3.0.0'
+  s.dependency 'Analytics', '~> 3'
```

Comment:
Sure, my comment was just to @f2prateek that it should not be `~> 3.0`, as I want it to be latest version, assuming it is compatible. I am fine with it being `~> 3.2` but of course the onus is ultimately on the pod maintainer to keep it up to date. In my case I had to discover the issue, open a ticket, and wait going on 2 weeks now for an update.